packages/wmr/src/wmr-middleware.js

```diff
@@ -39,7 +39,7 @@ export default function wmrMiddleware(options) {
 	distDir = resolve(dirname(out), distDir);
 
 	const NonRollup = createPluginContainer(getPlugins(options), {
-		cwd,
+		cwd: root,
```

Comment:
(meant to send this before) does this match what we initialize Rollup with in prod builds? It needs to match in order for paths to be the same I think (both will "work", but would generate different hashes).

Reply:
Just checked, there is no change in behaviour. Rollup itself doesn't seem to allow us to pass a custom `cwd` anywhere and here at this line we pass the same actual value as before. All plugins receive the same paths as before too.